cmd/utils/flags.go

```diff
@@ -321,6 +321,11 @@ var (
 		Name:  "ethash.dagslockmmap",
 		Usage: "Lock memory maps for recent ethash mining DAGs",
 	}
+	EthashEpochLengthFlag = cli.Int64Flag{
+		Name:  "epoch.length",
+		Usage: "Sets epoch length for makecache & makedag commands",
+		Value: 30000, // TODO - iquidus
```

Comment:
Do we want to add this value to default chain/ethash configs, or can we leave as is? (it's only used by makecache and makedag)

Reply:
Worth maybe exporting from the `ethash` package? 

Meh. I have no strong opinions, since it's pretty much set in stone and... just an adhoc constant for miscellaneous commands...

Reply:
Leaving as is for now.